reactioncommerce/reaction
# Navbar dropdowns conversion to React

Resolves:
#2044

**Description**
- The language and currency dropdown on the navbar should work exactly as they were in the Blaze implementation

**Testing**
- Click on the currency || language button on navbar
- Change language || currency
- Observe that there aren't any errors and that language || currency actually change
